### PR TITLE
change command names to prefix hc-, and updates all .json files with p…

### PR DESCRIPTION
**Pull request by philipbeadle**
…roject name

### CODE REVIEW

File: happs/create/default.nix [hunk 1]
```diff
@@ -1,6 +1,6 @@
 { pkgs }:
 let
-  name = "hn-happ-create";
+  name = "hc-happ-create";
```
Comment (robbiecarlton) [inline happs/create/default.nix:3]: the ticket says `hc-create-happ`
I slightly prefer that, but also see the utility in using `happ` as a prefix

**Review by robbiecarlton — CHANGES_REQUESTED** [review]
@philipbeadle - just one small change

File: happs/create/default.nix [hunk 2]
```diff
@@ -9,6 +9,13 @@ let
     tar -zxvf happ-template.tar.gz --strip-components=1 -C ./"''${1:-"Notes-hApp-Template"}"
     rm happ-template.tar.gz
     cd ''${1:-"Notes-hApp-Template"}
+    sed -i "s/react-graphql-template/"''${1:-"Notes-hApp-Template"}"/g" package.json
+    cd ui_src    
+    sed -i "s/react-graphql-template/"''${1:-"Notes-hApp-Template"}"/g" package.json
+    cd ..
+    cd dna_src    
+    sed -i "s/react-graphql-template/"''${1:-"Notes-hApp-Template"}"/g" app.json
+    cd ..
```
Comment (robbiecarlton) [inline happs/create/default.nix:18]: 👍

File: happs/create/default.nix (hunk 1 above)
Comment (philipbeadle) [reply happs/create/default.nix:3]: again it how david wants it so its consistent with the rest of the commands and follows the folder structure in holonix.

Comment (thedavidmeister) [reply happs/create/default.nix:3]: yeah it is chaos if you name things different to the folders

**Review by thedavidmeister — APPROVED** [review]

**Pull request merged**